GateWatcher/aioniq/_meta/fields.yml

```
@@ -384,10 +384,14 @@ gatewatcher.tlp:
   name: gatewatcher.tlp
   type: text
 
-gatewatcher.tls:
-  description: This field represents the tls field in a network metadata (used in
-    legacy format log)
-  name: gatewatcher.tls
+gatewatcher.tls.sni:
+  description: This field represents the TLS SNI field in a TLS metadata
+  name: gatewatcher.tls.sni
+  type: text
+
+gatewatcher.tls.fingerprint:
+  description: This field represents the TLS server fingerprint field in a TLS metadata
+  name: gatewatcher.tls.fingerprint
```

A text field cannot be transformed in a structured field. I suggest changing the name of the following fields to avoid conflict:
```suggestion
gatewatcher.tls_sni:
  description: This field represents the TLS SNI field in a TLS metadata
  name: gatewatcher.tls_sni
  type: text

gatewatcher.tls_fingerprint:
  description: This field represents the TLS server fingerprint field in a TLS metadata
  name: gatewatcher.tls_fingerprint
```